### ISSUE TITLE
Change prefix in learning rate logger from "lr-" to "lr/"

### ISSUE THREAD

**Issue by versatran01** [PyTorchLightning/pytorch-lightning#2097]
This allows all lr related values to be folded in tensorboard.

**Comment by Borda** [comment]
@SkafteNicki mind have look? :rabbit:

**Comment by SkafteNicki** [comment]
Fine by me, right now '-' is used to indicate the learning rates from different optimizers and '/' is used to indicate different param groups from the same optimizer. But I guess it is more common to have multi optimizer setup than multi param groups. So we can just change them around or make them both '/'.

@versatran01 would you be up for a PR, it mainly is a few lines in https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/callbacks/lr_logger.py that needs to be changed.

**Comment by versatran01** [comment]
@SkafteNicki I will take a stab at it.
